package.json

```diff
@@ -10,7 +10,10 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "ipad": "react-native run-ios --simulator=\"iPad Pro (12.9-inch) (4th generation)\"",
-    "desktop": "webpack --config config/webpack/webpack.dev.js --platform desktop && electron desktop/main.js",
+    "desktop": "webpack --config config/webpack/webpack.dev.js --platform desktop && npm run desktop:electron",
```

Comment:
Can we simplify this at all? Why not just have a single command for `npm run desktop`? Would you ever want to run any of these other scripts in isolation? I'd imagine not...

Reply:
Also, then we wouldn't need the extra `npm-run-all` dependency

Reply:
npm-run-all is used just to kill another process if one of them is crashed. It also lets us have categorized output.

Reply:
Okay, can we still clean this up so that all one needs to do to run the webpack dev server and electron app with hot reload is run `npm run desktop` ?

Reply:
sure